src/App.tsx

```diff
@@ -1,7 +1,51 @@
 import React from 'react';
 
+type State = {
+  pressedKey: boolean,
+  userPressed: string,
+};
+
+export class Click extends React.Component<{}, State> {
+  state: Readonly<State> = {
+    pressedKey: false,
+    userPressed: '',
+  };
+
+  componentDidMount() {
+    document.addEventListener('keyup', this.handleDocumentClick);
+  }
+
+  componentWillUnmount() {
+    document.removeEventListener('keyup', this.handleDocumentClick);
+  }
+
+  handleDocumentClick = (event: KeyboardEvent) => {
+    if (event.key) {
+      const userPressed1 = event.key;
```

Comment:
You assign name of the key to userPressed when you have variable called pressedKey, to me it would be more obvious when you would assign name of the key to pressedKey